### PR TITLE
Package rea.0.2.0

### PR DESCRIPTION
**Pull request by polytypic**
### `rea.0.2.0`
Effectful OCaml with Objects and Variants

---
* Homepage: https://github.com/polytypic/rea-ml
* Source repo: git+https://github.com/polytypic/rea-ml.git
* Bug tracker: https://github.com/polytypic/rea-ml/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/rea/rea.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+synopsis: "Effectful OCaml with Objects and Variants"
+maintainer: "Vesa Karvonen <vesa.a.j.k@gmail.com>"
+authors: "Vesa Karvonen"
+license: "MIT"
+homepage: "https://github.com/polytypic/rea-ml"
+bug-reports: "https://github.com/polytypic/rea-ml/issues"
+dev-repo: "git+https://github.com/polytypic/rea-ml.git"
+depends: [
+  "dune" {>= "3.3.0"}
+  "ocaml" {>= "4.08.0"}
```
Comment (polytypic) [inline packages/rea/rea.0.2.0/opam:11]: Hmm... Why is the CI failing with earlier OCaml versions?  With my previous 0.1.0 submission the earlier OCaml versions were skipped by the CI.

Comment (kit-ty-kate) [reply packages/rea/rea.0.2.0/opam:11]: That was my fault, it’s a change in CI which has been fixed

File: packages/rea/rea.0.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "ppx_optcomp" {>= "v0.14.0"}
+  "ocamlformat" {dev}
+  "odoc" {dev}
```
Comment (kit-ty-kate) [inline packages/rea/rea.0.2.0/opam:14]: These should not be dev dependencies. See https://github.com/ocaml/opam/issues/4959
```suggestion
```

Comment (polytypic) [reply packages/rea/rea.0.2.0/opam]: Thanks for the note!

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**